thechelle13/tech-client
# login and register functional in UI

# Login and Register Functionality

**Changes Made:**
- Implemented login and register functionality in the UI.
- Modified various files to incorporate login and registration features.
- Created new components: `Login`, `Register`, `NavBar`.
- Removed and renamed files related to authentication.

## STEPS TO TEST

1. Pull down the latest changes.
2. Start/Restart your application.
3. Navigate to the login and registration pages.
4. Verify that the login and registration functionality works as expected.
5. Test the UI for proper navigation after login.
6. Ensure that the UI reflects successful authentication.

# Note to Reviewers

- Please review the changes thoroughly, especially the new components and modifications related to authentication.
- Verify that the login and registration processes are working correctly.
- Check for any potential issues with navigation and user authentication.
